src/Symfony/Component/Form/ClearableErrorInterface.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form;
+
+/**
+ * A form element whose errors can be cleared.
+ *
+ * @author Colin O'Dell <colinodell@gmail.com>
+ */
+interface ClearableErrorInterface
+{
+    /**
+     * Removes all the errors of this form.
+     *
+     * @param bool $deep whether to remove errors from child forms as well
+     *
+     * @return $this
+     */
+    public function clearErrors($deep = false);
```

Comment:
Let's add type hint to argument `public function clearErrors(bool $deep = false);`.
But please don't add `: self`, we already had discussion on this (https://github.com/symfony/symfony/pull/27190#discussion_r186892825).

Reply:
Added and amended!  Thanks for the feedback :+1: